src/tooltips/TooltipWrapper.js

```diff
@@ -12,32 +12,32 @@ import DefaultTooltip from './Tooltip'
 
 const MARGIN = 15
 
-export function getTooltipPosition(mouse, height, width) {
+export function getTooltipTransform(mouse, height, width) {
```

Comment:
What is `height`/`width` in this context? What's the reasoning for using it instead of `innerHeight`/`innerWidth`?

Reply:
Yeah, I was confused about that too. It's the computed height/width of the tooltip itself, as measured by `BlockSize`. It would be nice to get rid of `BlockSize` and the `withControlledState` wrapper and have Tooltip manage its own state and layout computation for the sake of clarity.